_data/software/eddypro.yml

```diff
@@ -0,0 +1,36 @@
+---
+schemaVersion: '1.0'
+description:
+  whatItDoes:
+    en: >-
+      EddyPro® Software is a powerful, free, and open source application for
+      processing eddy covariance data.
+    fr: >-
+      Le logiciel EddyPro® est une application open source puissante et gratuite
+      pour le traitement des données de covariance des courants de Foucault.
+category: '43232600'
+homepageURL:
+  en: 'https://www.licor.com/env/products/eddy_covariance/software.html'
+  fr: 'https://www.licor.com/env/products/eddy_covariance/software.html'
+licences:
+  - URL:
+      en: 'https://github.com/LI-COR/eddypro-engine/blob/master/LICENSE'
+      fr: 'https://github.com/LI-COR/eddypro-engine/blob/master/LICENSE'
+    spdxID: GPL-3.0-only
```

Comment:
I get a little tripped up by the EULA stuff they mention here https://github.com/LI-COR/eddypro-engine/#license

Reply:
It's definitely any interesting one - but isn't a barrier under our contexts - but does violate the definition of Open Source that we've put.

https://github.com/LI-COR/eddypro-engine/blob/master/LICENSE.txt

Do you think this may be worth discussing as a community?

Reply:
Probably, or figure out a way to caveat the listing. Kinda like what happened with Reflector https://en.wikipedia.org/wiki/.NET_Reflector